src/languages/en.ts

```diff
@@ -1728,34 +1726,34 @@ export default {
         hasBeenThrottledError: 'An error occurred while adding your bank account. Please wait a few minutes and try again.',
         hasCurrencyError: 'Oops! It appears that your workspace currency is set to a different currency than USD. To proceed, please set it to USD and try again.',
         error: {
-            youNeedToSelectAnOption: 'You need to select an option to proceed.',
-            noBankAccountAvailable: 'Sorry, no bank account is available.',
-            noBankAccountSelected: 'Please choose an account.',
-            taxID: 'Please enter a valid tax ID number.',
-            website: 'Please enter a valid website. The website should be in lowercase.',
-            zipCode: `Incorrect zip code format. Acceptable format: ${CONST.COUNTRY_ZIP_REGEX_DATA.US.samples}`,
-            phoneNumber: 'Please enter a valid phone number.',
-            companyName: 'Please enter a valid legal business name.',
-            addressCity: 'Please enter a valid city.',
-            addressStreet: 'Please enter a valid street address that is not a PO box.',
-            addressState: 'Please select a valid state.',
-            incorporationDateFuture: 'Incorporation date cannot be in the future.',
-            incorporationState: 'Please select a valid state.',
-            industryCode: 'Please enter a valid industry classification code. Must be 6 digits.',
-            restrictedBusiness: 'Please confirm company is not on the list of restricted businesses.',
-            routingNumber: 'Please enter a valid routing number.',
-            accountNumber: 'Please enter a valid account number.',
-            routingAndAccountNumberCannotBeSame: 'The routing number and account number cannot be the same.',
-            companyType: 'Please select a valid company type.',
-            tooManyAttempts: 'Due to a high number of login attempts, this option has been temporarily disabled for 24 hours. Please try again later or manually enter details instead.',
-            address: 'Please enter a valid address.',
-            dob: 'Please select a valid date of birth.',
-            age: 'Must be over 18 years old.',
-            ssnLast4: 'Please enter valid last 4 digits of SSN.',
-            firstName: 'Please enter a valid first name.',
-            lastName: 'Please enter a valid last name.',
-            noDefaultDepositAccountOrDebitCardAvailable: 'Please add a default deposit bank account or debit card.',
-            validationAmounts: 'The validation amounts you entered are incorrect. Please double-check your bank statement and try again.',
+            youNeedToSelectAnOption: 'Please select an option to proceed',
```

Comment:
Why these copy changes?

Reply:
cc @jamesdeanexpensify

Reply:
This is weird - is the red area something that we would be removing? I don't think I would ever suggest those changes (removing lines entirely).

Reply:
Mmm, I don't remember at this point, but whatever is in there I got from that old f'ed up PR from a while back, I didn't make any changes directly to the copy. To clarify, that content is not being deleted, you can see that the updated lines are below: https://github.com/Expensify/App/pull/44651/files

Reply:
OK, I made some minor updates and I think we're good now! Let me know what you think.

Reply:
Looks good!